index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css">
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <nav id="navbar">
+        <ul>
+            <li>
+                <a href="#welcome-section">Welcome</a>
+            </li>
+            <li>
+                <a href="#projects">Check my projects</a>
+            </li>
+            <li> <a href="#contact">Contact me</a>
+            </li>
+        </ul>
+    </nav>
+    <section id="welcome-section">Welcome!
+        <H1> Welcome to my personal Portfolio</H1>

```

Comment:
Please use lowercase letters to name HTML tags. As it is a requirement.

Reply:
Well done on this. 

However, Your comment could be better put 👇🏾 

> Even though, HTML is not a Case Sensitive Language because, during parsing, all HTML elements are converted to lowercase first. Please I recommend you always use lowercase letters when naming HTML tags because it is industry standard and best practice.

This sound more professional and convincing. Kindly update your comment by giving more details.